### ISSUE TITLE
Graph in documentation

### ISSUE THREAD

**Issue by TWiStErRob** [google/dagger#1455]
Is this `dot` graph meant to be rendered?

https://github.com/google/dagger/blame/gh-pages/semantics/index.md#L300-L342

**Comment by ronshapiro** [comment]
Yep. We have something internally to do this, but we should probably
abandon that and generate the file and check that in.
